dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -171,118 +171,601 @@
      ],
      "rows": [
       {
-       "value": "I am going to love teachi",
+       "value": "I_am_going_to_love_teachi",
```

Comment:
do we want to have cut off words as the keys?

Reply:
also note the matrix rows values above don't have underscores yet

Reply:
Fixed.  Went through all surveys and set slugs by hand.